sbol_utilities/default_components.py

```diff
@@ -0,0 +1,148 @@
+from typing import List, Tuple
+
+import sbol3
+import tyto
+
+def promoter(name: str, description: str, sequence: str)-> Tuple[sbol3.Component, sbol3.Sequence]:
+    """
+    Creates a promoter and a sequence Components with a given name, description, and sequence.
+    """
+    promoter_seq = sbol3.Sequence(f'{name}_seq')
+    promoter_seq.elements= sequence
+    promoter_seq.encoding = 'https://identifiers.org/edam:format_1207'
```

Comment:
Use `sbol3.IUPAC_DNA_ENCODING`

Reply:
Thanks I could not found that, I will replace the URI